Update _index.md (#27734)
DataDog/documentation: content/en/ddsql_editor/_index.md

## Query in natural language

> Type your question into the search box, and Datadog builds the SQL query for you. [Reach out to support][5] to enable it if not already. 

Reviewer: ```suggestion
Type your question into the search box, and Datadog builds the SQL query for you. If you haven't already, [reach out to support][5] to enable this feature.
```
